### PR TITLE
rename SetAccountKey->UpdateAccountKey

### PR DESCRIPTION
**Pull request by Jinming-Hu**

### CODE REVIEW

File: sdk/storage/azure-storage-common/inc/azure/storage/common/storage_credential.hpp [hunk 1]
```diff
@@ -32,7 +32,7 @@ namespace Azure { namespace Storage {
     {
     }
 
-    void SetAccountKey(std::string accountKey)
+    void UpdateAccountKey(std::string accountKey)
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-common/inc/azure/storage/common/storage_credential.hpp:35]: I am assuming docs will be coming later for some of these?

**Review by ahsonkhan — APPROVED** [review]

File: sdk/storage/azure-storage-common/inc/azure/storage/common/storage_credential.hpp (hunk 1 above)
Comment (Jinming-Hu) [reply sdk/storage/azure-storage-common/inc/azure/storage/common/storage_credential.hpp:35]: yeah, we're making a lot of changes. I'm going to sum up and make up all the changelog before next release.

Comment (ahsonkhan) [reply sdk/storage/azure-storage-common/inc/azure/storage/common/storage_credential.hpp:35]: That works for the changelog.

I meant adding API doxygen docs (brief, param, return, exceptions, etc.).

Comment (Jinming-Hu) [reply sdk/storage/azure-storage-common/inc/azure/storage/common/storage_credential.hpp:35]: Well, I have no idea why it doesn't have doxygen docs.

Comment (Jinming-Hu) [reply sdk/storage/azure-storage-common/inc/azure/storage/common/storage_credential.hpp:35]: i'm going to add it right now

**Pull request merged**